python/graphstorm/gconstruct/file_io.py

```diff
@@ -118,7 +118,7 @@ def read_index_parquet(data_file, column):
 
     if len(column) == 1:
         res_array = df[column[0]].to_numpy()
-    elif len(df.columns) == 2:
+    elif len(column) == 2:
```

Comment:
What is this change for?

Reply:
Make this check consistent with the previous one, 

```python
    if len(column) == 1:
        res_array = df[column[0]].to_numpy()
    elif len(column) == 2:
```